_posts/sean-has-a-crystal-ball.md

```diff
@@ -0,0 +1,110 @@
+
+Confidence leads me to the second example. Every Italian has a thing called "Tessera sanitaria" (a Healthcare card, a sort of Social Security card provided by the healthcare system). In the region where I live, that's merged with another card, called "Tessera regionale dei servizi" (Regional services card). The Regional services card theoretically gives you access to additional services, such as document digital signing. When the pandemic was at its maximum effect at the beginning of last summer, I had to go through some Italian bureaucracy to sign some documentation.
+Given that most public offices were shut down, the only option was the "online" option. Unfortunately, the Regional services card needs to be activated for these additional services. To make a long story short, the process took me a little more than three months for a signing operation that lasted 5 seconds. The incompetence of the Italian public administration starts with little things like feedback to users. To enable the additional service, one needs to send an email with a bunch of documentation. Unfortunately, the email address behaves like Gargantua, the famous Interstellar movie black hole; there is zero feedback. There is no website one can go to to look up the status of their request. A month in, I got an email, obviously from a different email address, telling me that my request was incomplete. So let's start the process again, a new email, this time to a different email address with additional documentation and no way to get any feedback. "Simple affairs complication department" is the way many Italians refer to the public administration.
+
+We could say that the problem is that we're leaving the user groping in the dark with no way to understand what is going on. Under the hood is the consequence of dealing badly, or not dealing at all, with eventual consistency.
+
+In ["Ehi! What's up? Feedback to users' requests in distributed systems"](https://milestone.topics.it/2021/01/12/feedback-to-users-requests-in-distributed-systems.html) I describe the flow and the infrastructure required to provide feedback to users waiting for feedback. "Waiting for feedback" is the crucial aspect of the article. The sample mentioned above wouldn't benefit from the described approach; in both cases, we won't sit in front of the screen, steering at it, waiting for a status update. In the Amazon order sample, we know that it will take at least a few hours; in the Italian public administration case because we know it won't happen for weeks.
```

Comment:
```suggestion
In ["Ehi! What's up? Feedback to users' requests in distributed systems"](https://milestone.topics.it/2021/01/12/feedback-to-users-requests-in-distributed-systems.html) I describe the flow and the infrastructure required to provide feedback to users waiting for feedback. "Waiting for feedback" is the crucial aspect of the article. The sample mentioned above wouldn't benefit from the described approach; in both cases, we won't sit in front of the screen, staring at it, waiting for a status update. In the Amazon order sample, we know that it will take at least a few hours, and in the Italian public administration case it won't happen for weeks.
```